homeassistant/components/pvoutput/sensor.py

```diff
@@ -81,26 +111,44 @@ async def async_setup_entry(
 ) -> None:
     """Set up a Tailscale binary sensors based on a config entry."""
     coordinator = hass.data[DOMAIN][entry.entry_id]
-    async_add_entities([PvoutputSensor(coordinator)])
+    async_add_entities(
```

Comment:
Is the description above still correct?
`"""Set up a Tailscale binary sensors based on a config entry."""`

Reply:
Nope, adjusted!